x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -286,7 +286,7 @@ export const DataFrameAnalyticsList: FC<Props> = ({
               <EuiFlexItem grow={false}>
```

Comment:
`{!isManagementTable && <EuiSpacer size="m" />}` on line 278 can go now. it adds unnecessary white space to the top of the page